### ISSUE TITLE
Parrot install for debian not working:

### ISSUE THREAD

**Issue by ViSioN-BluR** [ParrotSec/alternate-install#1]
root@ViSioN-BluR:~/parot# wget https://raw.githubusercontent.com/ParrotSec/alternate-install/master/parrot-install.sh
--2015-04-10 09:15:36--  https://raw.githubusercontent.com/ParrotSec/alternate-install/master/parrot-install.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 199.27.78.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|199.27.78.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 4362 (4.3K) [text/plain]
Saving to: `parrot-install.sh'

100%[=============================================================================================================================>] 4,362       --.-K/s   in 0.004s  

2015-04-10 09:15:38 (1012 KB/s) - `parrot-install.sh' saved [4362/4362]

root@ViSioN-BluR:~/parot# chmod +x parrot-install.sh 
root@ViSioN-BluR:~/parot# ./parrot-install.sh 
./parrot-install.sh: line 111: syntax error near unexpected token `)'
./parrot-install.sh: line 111: `	6) clear;'
root@ViSioN-BluR:~/parot#

**Comment by ViSioN-BluR** [comment]
I did everything manualy :) to make it work

**Comment by EclipseSpark** [comment]
problem identified, there is a missing double semicolon

**Comment by ViSioN-BluR** [comment]
Yup i know, it didn't work propely there script and there ParrotSec OS has alot problems too, if you have irc server or something for support like live chat please tell me i got alot issue's which i needed fix manualy..by my self :)

**Comment by EclipseSpark** [comment]
you can join our irc channel on freenode (#frozenbox)
